File: 981-delete-columns-to-make-sorted/delete-columns-to-make-sorted.cpp
class Solution {
public:
    int minDeletionSize(vector<string>& strs) {
        int n = strs.size();
        int count =0;
        int m = strs[0].size();
        int idx =0;
        for(int idx =0 ;idx<m;idx++){
            char c = strs[0][idx];
            for(int i =1;i<n;i++){
                if(c>strs[i][idx]){
                    count++;
                    break;
                }
                c = strs[i][idx];

            }
        }
        return count;
        
    }
};

File: 981-delete-columns-to-make-sorted/delete-columns-to-make-sorted.java
class Solution {
    public int minDeletionSize(String[] strs) {
        int n = strs.length;
        int count =0;
        int m = strs[0].length();
        for(int idx =0;idx<m;idx++){
            char c = strs[0].charAt(idx);
            for(int i =1;i<n;i++){
                if(c>strs[i].charAt(idx)){
                    count++;
                    break;
                }
                c = strs[i].charAt(idx);
            }
        }
        return count;
    }
}